src/lib/components/drs/formComponents/DrsObjectParents.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import DrsObjectRelatives from './DrsObjectRelatives';
+import { 
+    Typography
+} from '@material-ui/core';
+
+const DrsObjectParents = props => {
+    const assertParentIsBundleThenSetValid = (responseData) => {
```

Comment:
Currently, the DrsObjectParents are set as invalid regardless of the type of object that is set as the parent (i.e. both blob and bundle objects are set as invalid). DrsObjectChildren are verified correctly, however, any ID that is entered as a DrsObjectParent is marked as invalid.